ARGO/src/ParticleSystem.cpp

```diff
@@ -0,0 +1,71 @@
+#include "stdafx.h"
+#include "ParticleSystem.h"
+#include "ext.hpp"
+
+ParticleSystem::~ParticleSystem()
+{
+}
+
+void ParticleSystem::update(Entity& t_entity)
+{
+	if (t_entity.getAllComps().at(COMPONENT_ID::PARTICLE_ID) && t_entity.getAllComps().at(COMPONENT_ID::PRIMITIVE_ID) && t_entity.getAllComps().at(COMPONENT_ID::TRANSFORM_ID))
+	{
+		ParticleEmitterComponent* partComp = static_cast<ParticleEmitterComponent*>(t_entity.getAllComps().at(COMPONENT_ID::PARTICLE_ID));
```

Comment:
ambiguous name. kinda makes it seems like its part of a component

Reply:
Adjusted.